src/utils/restUtils.getUrl.test.ts

```diff
@@ -288,7 +288,7 @@ describe("isDevSbs", () => {
     }
 });
 
-describe("isDevGcp", () => {
+describe("isDevGcpWithProxy", () => {
```

Comment:
Her er tittel `isDevGcpWithProxy`, mens på linje 319 ser det ut som det er `isDevGcpWithoutProxy` som blir testet? 

Mangler tester for `isDevGcpWithProxy`?

Reply:
Det gjorde den. Har oppdatert testene nå.
Lurer litt på på om det virkelig er nødvendig med så mye URL tester.